src/fiatconnect/FiatDetailsScreen.tsx

```diff
@@ -329,49 +359,34 @@ const FiatDetailsScreen = ({ route, navigation }: Props) => {
             </View>
           )
         })}
-      </ScrollView>
-
-      <View style={styles.footer}>
-        <View style={styles.paymentOption}>
-          <BorderlessButton onPress={onPressSelectedPaymentOption} testID="selectedProviderButton">
-            <View style={styles.paymentOptionButton}>
-              <Text style={styles.paymentOptionText}>
-                {t('fiatDetailsScreen.selectedPaymentOption')}
-              </Text>
-              <ForwardChevron color={colors.gray4} />
-              <Image
-                source={{
-                  uri: quote.getProviderLogo(),
-                }}
-                style={styles.iconImage}
-                resizeMode="contain"
-              />
-            </View>
-          </BorderlessButton>
-        </View>
-        <View>
-          <Button
-            testID="nextButton"
-            text={t('next')}
-            onPress={onPressNext}
-            disabled={!validInputs}
-            style={styles.nextButton}
-            size={BtnSizes.FULL}
-          />
-        </View>
-      </View>
+      </KeyboardAwareScrollView>
```

Comment:
I haven't studied the `KeyboardAwareScrollView` and `KeyboardSpacer` components, but is the general idea that `KeyboardAwareScrollView` grows to fit the available space when the keyboard isn't shown, and shrinks up to a max of the keyboard height when it's shown? And `KeyboardSpacer` basically does the opposite (i.e., invisible by default, but expands to the height of the keyboard when it's shown, thus "pushing" the button up)? Pretty cool!

Reply:
the `KeyboardSpacer` is the one that animates when the keyboard comes in / goes out of view. the `KeyboardAwareScrollView` ensures the text field on focus is visible in scroll when the keyboard comes in to view.